app/assets/stylesheets/application.scss

```diff
@@ -185,6 +185,14 @@
   @include heading-36;
 }
 
+.heading-24 {
+  @include heading-24;
+}
+
+.copy-19 {
+  @include copy-19;
+}
```

Comment:
Are these required they seem to create the same class name? also these are terrible class names

Reply:
I've renamed these classes, agreed they were not good names